YourFirstPR.md

```diff
@@ -0,0 +1,77 @@
+# Your first PR
+
+## Finding things to work on
+
+The core team usually tags issues that are ready to be worked on and easily accessible for new contributors with the [“you can do this” label][you can do this]. If you’ve never contributed to _fastlane_ before, these are a great start!
+
+If you want to work on something else, e.g. new functionality or fixing a bug, we kindly ask you to submit a new issue, so that we can have a chance to discuss it first. We might have some pointers for you on how to get started, or how to best integrate it with existing solutions.
+
+## Checking out the fastlane repo
+
+- Click the “fork” button in the upper right corner of the [main fastlane repo][fastlane]
+- Clone your fork:
+  - `git clone <YOUR_GITHUB_USER> git@github.com:<YOUR_GITHUB_USER>/fastlane.git`
+- Install dependencies:
+  - Run `bundle install` in the project root
+  - You also might need to run `bundle install` in each of the tool's subdirectories, e.g. `cd gym && bundle install`
+- Create a new branch to work on:
+  - `git checkout -b <YOUR_BRANCH_NAME>`
+  - A good name for a branch describes the thing you’ll be working on, e.g. `docs-fixes`, `fix-deliver-upload`, `gym-build-android-app`, etc.
+- That’s it! Now you’re ready to work on fastlane 
+
+## Testing your local changes
+
+Make sure to run the automated tests using `bundle exec` to ensure you’re running the correct version of `rspec` and `rubocop`
+
+First, navigate into the sub-directory of the tool you want to test.
+
+To run the automated unit tests
+
+```
+bundle exec rspec
+```
+
+To verify and auto-fix the code style
+
+```
+bundle exec rubocop -a
+```
+
+After introducing some changes to the _fastlane_ source code, you probably want to test the changes for your application. The easiest way to do so it use [bundler](https://bundler.io/).
+
+Copy the Gemfile [.assets/Gemfile](.assets/Gemfile) from your local fastlane clone and drop it into your project's root folder.
```

Comment:
> Now that all sub-projects refer to projects' source code (instead of Gems) by default, is this instruction still necessary?

By "your project's root folder" we are talking about the user's test project, where they want to test out there changes to fastlane.